tests/AdagucTests/TestWMSVolScan.py

```diff
@@ -0,0 +1,85 @@
+import os
+import os.path
+from io import BytesIO
+import unittest
+import shutil
+import subprocess
+import json
+from lxml import etree, objectify
+import re
+from adaguc.AdagucTestTools import AdagucTestTools
+from lxml import etree, objectify
+import datetime
+
+ADAGUC_PATH = os.environ['ADAGUC_PATH']
+
+
+class TestWMSVolScan(unittest.TestCase):
```

Comment:
Can you also add a testcase for getlegendgraphic? The autowms getlegendgraphic on volume scans crashes